chia/simulator/block_tools.py

```diff
@@ -1870,7 +1867,13 @@ def load_block_list(
             sp_hash = full_block.reward_chain_block.challenge_chain_sp_vdf.output.get_hash()
 
         cache = BlockCache(blocks)
-
+        prev_transaction_b_height = pre_sp_block_height(
```

Comment:
I think you should remove the variable declaration on line 1853 as well. Maybe even rename this variable to `prev_tx_height`